public/data/HimanshuSinghNegi.json

```
@@ -0,0 +1,32 @@
+{
+  "name": "Himanshu Singh Negi",
+  "type": "personal",
+  "bio": "A student trying to learn new things daily.",
+  "avatar": "https://github.com/HimanshuSinghNegi.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/HimanshuSinghNegi",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/Hnegi2980Negi",
+      "icon": "twitter"
+    },
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/himanshusinghnegi",
+      "icon": "linkedin"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Open Souce Contribution",
+      "date": "October,14 2022",
```

You might want to fix this minor issue here.
```suggestion
      "date": "October 14, 2022",
```